src/PantheonTerminalWindow.vala

```diff
@@ -157,6 +161,9 @@ namespace PantheonTerminal {
                 app.set_accels_for_action (ACTION_PREFIX + action, action_accelerators[action].to_array ());
             }
 
+            app.set_accels_for_action (ACTION_PREFIX + ACTION_SEARCH_NEXT, {"<Control>g", "<Control>Down"});
+            app.set_accels_for_action (ACTION_PREFIX + ACTION_SEARCH_PREVIOUS, {"<Control><Shift>g", "<Control>Up"});
```

Comment:
Why not extend the action_accelerators multimap?

Reply:
Just seemed like a lot more work :grimacing: